netlogo-gui/src/main/api/TokenHolder.scala

```diff
@@ -4,6 +4,7 @@ package org.nlogo.api
 
 import org.nlogo.core.Token
 
+@deprecated("6.0.3", "User core.TokenHolder instead")
```

Comment:
Typo, should be "Use core.TokenHolder instead" (Use, not User)

Reply:
Nice catch!